backend/audit/intakelib/checks/check_version_number.py

```diff
@@ -18,6 +18,7 @@
     "1.1.1",
     "1.1.2",
     "1.1.3",
+    "1.1.4",
```

Comment:
@phildominguez-gsa do the updated templates need to be published to the static site before this PR merges?

Reply:
Oh of course! Thanks, James.